components/collector/tests/source_collectors/quality_time/test_missing_metrics.py

```diff
@@ -90,3 +90,11 @@ async def test_nr_of_missing_metrics_without_correct_report(self):
         self.reports["reports"] = []
         response = await self.collect(get_request_json_side_effect=[self.data_model, self.reports])
         self.assert_measurement(response, parse_error="No reports found with title or id")
+
+    async def test_subjects_to_ignore(self):
+        """Test that the number of non-ignored missing metrics is returned."""
+        self.set_source_parameter("subjects_to_ignore", ["S2"])
+        response = await self.collect(get_request_json_side_effect=[self.data_model, self.reports])
+        self.assert_measurement(
+            response, value=str(int(len(self.entities) / 2)), total=self.expected_software_metrics  # TODO - total/2?
```

Comment:
Why the TODO? The total doesn't change if the user decides to ignore subjects, so the TODO is not needed, I'd say.

Reply:
That was also what I thought, but I wanted to double-check :-)